src/main/java/frc/robot/commands/drivetrain/vision/AlignScore.java

```diff
@@ -43,20 +39,18 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    angle = isVisionBased ? getVisionBasedAngle() : getPositionBasedAngle();
-
-    if (Math.abs(angle) < 20 && turn > 0) {
-      rightX = 0.7 * angle / 100.0;
-    } else if (turn < 0) {
-      rightX = 0.25;
+    angle = isVisionBased ? getVisionBased() : getPositionBased();
+    
+    if (Math.abs(angle) < 10) {
+      rightX = VisionConstants.simpleAlignYInput * angle / 100.0; // Formula for turn value when the distance to angle is less than 10 degrees
+    } else if (Math.abs(angle) < 20) {
+      rightX = angle < 0 ? -.15 : .15;  // Turn value when the distance to angle is less than 20 degrees
+    } else {
+      rightX = angle < 0 ? -.23 : .23; // Turn value when the distance to angle is greater than 20 degrees
```

Comment:
this makes sense, good comments to, but for floating point numbers, usually put a `0` before them, make it a little easier to read and not miss the `.`
```suggestion
      rightX = angle < 0 ? -0.15 : 0.15;  // Turn value when the distance to angle is less than 20 degrees
    } else {
      rightX = angle < 0 ? -0.23 : 0.23; // Turn value when the distance to angle is greater than 20 degrees
```
Once we know if this works this can be extracted into a function probably, or use a switch statement.